packages/test/src/init-fixture.js

```diff
@@ -1,11 +1,11 @@
 import tempy from "tempy";
-import copyFixture from "./copyFixture";
-import gitInit from "./gitInit";
+import copyFixture from "./copy-fixture";
+import gitInit from "./git-init";
 
-export default initFixture;
+module.exports = initFixture;
```

Comment:
Why `module.exports` instead of `export default`?

Reply:
Moving this to a separate package disables jest's automatic compilation for the sources, thus ES2015 modules are unavailable.

I aimed at making this work for Node v8 for this draft, we will have to come up with a babel build for this package eventually